src/app/components/fileupload/fileupload.ts

```diff
@@ -826,12 +833,22 @@ export class FileUpload implements AfterViewInit, AfterContentInit, OnInit, OnDe
      * @param {Number} index - Index of the file to be removed.
      * @group Method
      */
-    removeUploadedFile(index) {
+     removeUploadedFile(index) {
         let removedFile = this.uploadedFiles.splice(index, 1)[0];
         this.uploadedFiles = [...this.uploadedFiles];
         this.onRemoveUploadedFile.emit({ file: removedFile, files: this.uploadedFiles });
     }
 
+    /**
+     * Cancel upload file request.
+     * */
+    cancelUploadRequst() {
```

Comment:
typo error, should be `cancelUploadRequest`

Reply:
Thx, I fixed it.